src/main/interface/cli.c

```diff
@@ -3978,22 +3978,23 @@ static void printResource(uint8_t dumpMask)
 
         for (int index = 0; index < MAX_RESOURCE_INDEX(resourceTable[i].maxIndex); index++) {
             const ioTag_t ioTag = *((const uint8_t *)currentConfig + resourceTable[i].stride * index + resourceTable[i].offset);
-            const ioTag_t ioTagDefault = *((const uint8_t *)defaultConfig + resourceTable[i].stride * index + resourceTable[i].offset);
+            ioTag_t ioTagDefault = NULL;
```

Comment:
`DEFIO_TAG__NONE` or `IO_TAG(NONE)`?

Reply:
I actually prefer `NULL` here, as the case at hand is not one of 'the default tag is not defined', but of 'default in itself is not defined, and `ioTagDefault` should never be accessed as anything in this case' - the overlap between `NULL` and `IO_TAG(NONE)` just makes it impossible to check for this directly.